### DISCUSSION TITLE
Debugger key bindings

### DISCUSSION THREAD

**Discussion by wqweto** [twinbasic/twinbasic#1101] [General]
What is the current stance of debugger shortcuts? Do these emulate VB6 (`F8` and `Shift+F8`) or recent versions of Visual Studio (`F10` and `F11`)?

I'm (badly) missing `Ctrl+F10` for "Run to cursor" and `Ctrl+Shift+F9` for "Remove all breakpoints" to become productive.

The `Shift+F9` for "Quick watch" I'm mostly using to just press Enter key to execute default `Add` button which adds currently selected expression to Watches pane (and shows it if hidden).

cheers,
</wqw>

**Comment by fafalone** [comment]
Ideally you should be able to define your own shortcuts. Visual Studio has a keyboard shortcuts scheme list where you can choose to use VB6 shortcuts, VS6 shortcuts, VS Code shortcuts, etc. Even better would be loading custom ones.

A powerful IDE is really a must in the end, especially if you're trying to draw in people used to VB6 and VS IDEs.
